src/support/qsort_r.c

```diff
@@ -0,0 +1,10 @@
+/*-
+ * Copyright (c) 2014-present MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+
```

Comment:
Probably want to add a comment in here explaining where the work is being done - very clever.

Reply:
Fair enough - done in c406025, hopefully that helps.